bane-of-wargs.html

```diff
@@ -0,0 +1,26 @@
+<html>
+<link rel="stylesheet" type="text/css" href="style.css">
+	<title>Bane Of Wargs</title>
+	<body>
+	  <div class="header">
+	    <a href="index.html">Home</a> | <a href="running.html">Running Dungeons of Kathallion</a> | <a href="development.html">Development</a> | <span class="selected">Bane Of Wargs</span>
+	  </div>
+	  <div class="inner">
+
+    <img src="bane of wargs.png" width="620" />
+      
+    <p>What is BANE OF WARGS?</p>
+	<p>Bane Of Wargs is at first a game engine, that you can use to create any story, item, mount, weapon, villages or anything you'd imagine. It is free, and open-source, you can fork the game engine and tweak as you would like, to make it like you'd like!</p>
```

Comment:
```suggestion
	<p>Bane Of Wargs was at first a game engine, that you could use to create any story, item, mount, weapon, village, or anything you could imagine. It is free, and open-source; you can fork the game engine and tweak it as you like, to make it like you'd like!</p>
```

Reply:
Why writing it at past?

Reply:
Because of the "at first".
```suggestion
	<p>Bane Of Wargs is a game engine that you can use to create any story, item, mount, weapon, village or anything you can imagine. It is free, and open-source, and you can fork the game engine and tweak as you would like, to make it like you'd like!</p>
```
How's this, then.

Reply:
Yeah. Good point.